.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like the file is still checked in?

Reply:
Unfortunately, although you've now added this folder to `.gitignore`, so that no new changes to it will be tracked, the file is still present in the repo. Use `git rm .vscode/launch.json` to remove it, and then commit that change.